client/my-sites/site-settings/seo-settings/help.jsx

```diff
@@ -23,23 +26,26 @@ const SeoSettingsHelpCard = ( {
 
 	return (
 		<div>
-			<SectionHeader label={ translate( 'Search Engine Optimization' ) } />
-			<Card>
-				<p>
-					{ translate(
-						'{{b}}WordPress.com has great SEO{{/b}} out of the box. All of our themes are optimized ' +
-						'for search engines, so you don\'t have to do anything extra. However, you can tweak ' +
-						'these settings if you\'d like more advanced control. Read more about what you can do ' +
-						'to {{a}}optimize your site\'s SEO{{/a}}.',
-						{
-							components: {
-								a: <a href={ seoHelpLink } />,
-								b: <strong />
+			<SectionHeader label={ translate( 'Search engine optimization' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'Search Engine Optimization' )` *ES Score: 21*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/seo-settings-help-text-nudge-conflict)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).